<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastClear</class>
 <widget class="QMainWindow" name="FastClear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Fast Clear</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="contents">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="left_menu">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(152, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btn_menu">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>251</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	text-align: left;
	padding: 8px;
}
QPushButton:hover {
	color: rgb(0, 0, 0);
	background-color: rgb(168, 73, 73);
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: rgb(168, 90, 90);
}</string>
          </property>
          <property name="text">
           <string>    MENU</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Models/Images/menu.ico</normaloff>../Models/Images/menu.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_home">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>251</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	text-align: left;
	padding: 8px;
}
QPushButton:hover {
	color: rgb(0, 0, 0);
	background-color: rgb(168, 73, 73);
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: rgb(168, 90, 90);
}</string>
          </property>
          <property name="text">
           <string>    HOME</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Models/Images/home.ico</normaloff>../Models/Images/home.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_fast_clear">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>130</y>
            <width>251</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	text-align: left;
	padding: 8px;
}
QPushButton:hover {
	color: rgb(0, 0, 0);
	background-color: rgb(168, 73, 73);
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: rgb(168, 90, 90);
}</string>
          </property>
          <property name="text">
           <string>    FAST CLEAR</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Models/Images/sweeping.ico</normaloff>../Models/Images/sweeping.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_exit">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>190</y>
            <width>251</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	text-align: left;
	padding: 8px;
}
QPushButton:hover {
	color: rgb(0, 0, 0);
	background-color: rgb(168, 73, 73);
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: rgb(168, 90, 90);
}</string>
          </property>
          <property name="text">
           <string>    EXIT</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Models/Images/logout.ico</normaloff>../Models/Images/logout.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="home">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(67, 67, 67);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>240</y>
            <width>571</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Wide Latin</family>
            <pointsize>36</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>true</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	text-align: center;
	color: rgba(255, 255, 255, 180);
}</string>
          </property>
          <property name="text">
           <string>TRICKSTER</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>297</y>
            <width>571</width>
            <height>10</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>System</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	text-align: center;
	color: rgba(255, 255, 255, 180);
}</string>
          </property>
          <property name="text">
           <string>SYSTEM</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
         <widget class="QFrame" name="fast_clear">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>750</width>
            <height>600</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(67, 67, 67);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QComboBox" name="combo_box">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>141</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(213, 213, 213);</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_path">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>40</y>
             <width>411</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(213, 213, 213);</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_search_path">
           <property name="geometry">
            <rect>
             <x>580</x>
             <y>40</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	color: rgb(0, 0, 0);
	background-color: rgb(80, 80, 80);
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: rgb(100, 100, 100);
}</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_start">
           <property name="geometry">
            <rect>
             <x>630</x>
             <y>40</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	color: rgb(0, 0, 0);
	background-color: rgb(80, 80, 80);
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: rgb(100, 100, 100);
}</string>
           </property>
           <property name="text">
            <string>START</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QListWidget" name="lbl_log">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>79</y>
             <width>730</width>
             <height>510</height>
            </rect>
           </property>
           <property name="cursor" stdset="0">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Type clear:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>20</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
